src/test/java/org/folio/service/logs/ErrorLogServiceUnitTest.java

```diff
@@ -207,52 +205,14 @@ void saveWithAffectedRecord_shouldSaveWithHoldingsAndItemsInfo(VertxTestContext
   void saveWithAffectedRecord_shouldSaveWithHoldingsInfo(VertxTestContext context) {
     // given
     JsonObject instanceObject = new JsonObject();
-    instanceObject.put("instance", instance.getJsonObject("instance"));
-    instanceObject.put("holdings", instance.getJsonArray("holdings"));
-    when(errorLogDao.save(any(ErrorLog.class), eq(TENANT_ID))).thenReturn(succeededFuture(errorLog));
-    // when
-    Future<ErrorLog> future = errorLogService.saveWithAffectedRecord(instanceObject, ERROR_REASON, JOB_EXECUTION_ID, TENANT_ID);
-    // then
-    future.onComplete(ar -> context.verify(() -> {
-      assertTrue(ar.succeeded());
-      verify(errorLogDao).save(errorLogCaptor.capture(), eq(TENANT_ID));
-      ErrorLog errorLog = errorLogCaptor.getValue();
-      assertErrorLogWithHoldingsAndItems(errorLog, false);
-      context.completeNow();
-    }));
-  }
-
-  @Test
-  void saveWithAffectedRecord_shouldSaveWithHoldingsInfoWithoutItemIfHoldingIdIsEmpty(VertxTestContext context) {
```

Comment:
WHy is this test case removed, it should still pass right?

Reply:
removed check when item holdingRecordId equals to holding, as we do it in fetchHoldingsAndItems() method while building json for mapping process, so we don`t need this test